### DISCUSSION TITLE
CSS

### DISCUSSION THREAD

**Discussion by cesar-sanjuan** [globaleaks/globaleaks-whistleblowing-software#3443] [General]
Good morning
Is the upload of css files possible from the interface or does it have to be done on the server?
We only have one channel.
How do we have to call the .css styles.css? styles.min.css?

Thank you

**Comment by evilaliv3** [comment]
Hello! thank you for your question.

we confirme that it is possible to load a css file to customize the platform.

You find this functionality in "Settings", "Theme Customization"
